data/strings/strings.txt

```diff
@@ -16383,44 +16383,48 @@
 
   [editor_edits_sent_message]
     tags = ios
-    en = Your suggested changes have been sent
-    af = U voorgestelde veranderinge is verstuur
-    ar = لقد تم إرسال التغييرات التي اقترحتها
-    be = Прапанаваныя вамі змены былі адпраўлены
-    bg = Предложените от вас промени са изпратени
-    ca = S'han enviat els canvis que heu suggerit
-    cs = Odeslali jsme Vámi navržené změny
-    da = Dine foreslåede ændringer er blevet sendt
-    de = Ihre Änderungsvorschläge wurden gesendet
-    el = Οι προτεινόμενες αλλαγές σας έχουν σταλεί
-    es = Se han enviado sus cambios sugeridos
-    et = Sinu soovitatud muudatused on saadetud
-    eu = Iradokitako aldaketak bidali dira
-    fa = تغییر پیشنهادی شما ارسال شد
-    fi = Ehdottamasi muutokset on lähetetty
-    fr = Vos modifications suggérées ont été envoyées
-    hu = Javasolt változtatásainak elküldése sikerült
-    id = Saran perubahan Anda telah dikirim
-    it = Le modifiche suggerite sono state inviate
-    ja = あなたの提案の変更が送信されました
-    ko = 귀하의 제안 변경사항이 전송되었습니다
-    lt = Jūsų pasiūlyti pakeitimas išsiųsti
-    mr = तुम्ही सुचवलेले बदल पाठवले आहेत
-    nb = De foreslåtte endringene er sendt
-    nl = Uw voorgestelde wijzigingen zijn verzonden
-    pl = Twoje sugestie zostały wysłane
-    pt = As suas alterações sugeridas foram enviadas
-    pt-BR = Suas sugestões de mudança foram enviadas
-    ro = Modificările sugerate au fost trimise
-    ru = Предложенные вами изменения отправлены
-    sk = Vykonali sa vami odporúčané zmeny
-    sv = Dina föreslagna ändringar har skickats
-    th = คำแนะนำการเปลี่ยนแปลงของคุณถูกส่งไปแล้ว
-    tr = Önerdiğiniz değişiklikler gönderildi
-    uk = Зміни, що ви запропонували, відправлено
-    vi = Những thay đổi đề nghị của bạn đã được gửi
-    zh-Hans = 您的建议已发送
-    zh-Hant = 您的建議已傳送
+    en = Your note will be sent to OpenStreetMap
+    af = Jou nota sal na OpenStreetMap gestuur word
+    ar = سيتم إرسال ملاحظتك إلى OpenStreetMap
+    be = Ваша нататка будзе адпраўлена ў OpenStreetMap
+    bg = Вашата бележка ще бъде изпратена до OpenStreetMap
+    ca = La teva nota s'enviarà a OpenStreetMap
+    cs = Vaše poznámka bude odeslána do OpenStreetMap
+    da = Din note vil blive sendt til OpenStreetMap
+    de = Ihre Notiz wird an OpenStreetMap gesendet
+    el = Η σημείωσή σας θα σταλεί στο OpenStreetMap
+    es = Tu nota se enviará a OpenStreetMap
+    es-MX = Tu nota será enviada a OpenStreetMap
```

Comment:
Let's remove the es-MX variant and edit es as follows: "Su nota se enviará a OpenStreetMap".

Reply:
updated